src/core/views/mobile-modal.js

```diff
@@ -18,17 +18,17 @@ import {
 import { DEFAULT_AUTOCLOSE_TIMEOUT } from './popup'
 
 const Wrapper = styled.div`
+  height: ${({ height }) => (height ? height : '100%')};
   visibility: ${(p) => (p.open ? 'visible' : 'hidden')};
   opacity: ${(p) => (p.open ? 1 : 0)};
   background-color: ${(p) => p.backgroundColor};
   position: fixed !important;
   z-index: 10001;
   left: 0 !important;
   right: 0 !important;
-  top: 0 !important;
+  top: 80px !important;
```

Comment:
Why change of `top` when the tab bar seems to be at the bottom?

Reply:
Ooops, this was done to emulate the same situation from the issue, undone